src/page/blog/article-page.jsx

```diff
@@ -1,125 +1,250 @@
-        <Form.Submit
-          className="font-bold font-arial bg-white text-lg/4 text-black w-full border border-black py-4 px-4"
-          title={"Save to drafts"}
-          onClick={() => console.log("drafts...")}
-        />
-        <Form.Submit
-          className="font-bold font-arial bg-yellow-400  text-lg/4 text-black w-full border border-black py-4 px-4"
-          title={"Publish! "}
-        />
+      {!successful ? (
+        <div className="relative flex flex-row justify-center mx-auto -top-5 pt-20">
+          <ToastContainer className={"absolute"} />
+          <Form className={"w-[700px] ml-[224px] p-5 bg-white"}>
+            <Form.TextInput
+              className="relative flex justify-center font-bold font-arial flex-col py-3"
+              labelText={"Title"}
+              value={formState.title}
+              onChange={(e) =>
+                setFormState({ ...formState, title: e.target.value })
+              }
+              isValid={checkValidity(formState.title)}
+            />
+            <Form.TextArea
+              labelText={"Body"}
+              placeholder={`Body`}
+              value={formState.content}
+              onChange={(e) =>
+                setFormState({ ...formState, content: e.target.value })
+              }
+            />
+            <Form.TextInput
+              className=" flex font-bold font-arial flex-col py-3"
+              labelText={"Slops"}
+              placeholder={"Add topical slops"}
+              value={formState.movies.join(", ")}
+              onChange={(e) =>
+                setFormState({
+                  ...formState,
+                  movies: e.target.value.split(",").map((item) => item.trim()),
+                })
+              }
+              isValid={checkValidity(formState.movies.join(", "))}
+            />
+            <Form.TextInput
+              className=" flex font-bold font-arial flex-col py-3"
+              labelText={"Keywords"}
+              placeholder={"Add topical keywords"}
+              value={formState.keywords.join(", ")}
+              onChange={(e) =>
+                setFormState({
+                  ...formState,
+                  keywords: e.target.value
+                    .split(",")
+                    .map((item) => item.trim()),
+                })
+              }
+              isValid={checkValidity(formState.keywords.join(", "))}
+            />
+          </Form>
+          <div className="self-center mt-32 h-[49px] min-w-[224px] md:absolute md:bottom-0 md:left-0 md:right-0 md:top-96 xs:absolute xs:bottom-0 xs:left-0 xs:right-0 xs:top-80">
+            <Form onSubmit={onDraft}>
+              <Form.Submit
+                className="font-bold font-arial bg-white text-lg/4 text-black w-full border border-black py-4 px-4"
+                title={"Save to drafts"}
```

Comment:
Cannot submit draft unless both title and content are somewhat filled. Draft would imply that the user could submit a draft with one or the other. This should be fine for now and could be an improvement later on.